tests/system/providers/amazon/aws/example_sagemaker.py

```diff
@@ -159,12 +159,11 @@ def _build_and_upload_docker_image(preprocess_script, repository_uri):
         docker_build_and_push_commands = f"""
             cp /root/.aws/credentials /tmp/credentials &&
             # login to public ecr repo containing amazonlinux image
-            docker login --username {creds.username} --password {creds.password} public.ecr.aws
+            docker login --username {creds.username} --password {creds.password} public.ecr.aws &&
```

Comment:
the old version was working too, but at least this homogeneous with the other lines of the command